src/main/java/org/apache/mesos/scheduler/plan/PlanManager.java

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
There are three major themes:
1. External entity generates a Plan, and instantiates PlanManager with it (DeployManager). And, this plan doesn't change ever for the life-cycle of PlanManager. `setPlan` is not needed.
2. PlanManager generates it's own Plan (SimpleRecoveryPlanManager). `setPlan` is not needed.
3. An external entity generates plan at various points, and provides PlanManager with a new one whenever a new one is generates. (No such implemenation yet). `setPlan` is needed. 

Because we don't have use-case 3 right now, I would say we don't need `setPlan`. Thoughts ?